tools/livegraph/README.md

```diff
@@ -0,0 +1,8 @@
+# Livegraph
+
+This tool repeatedly plots the last given amount of lines from a preCICE conversion or iteration file.
+
+For more information please see the help:
+```
+livegraph.py --help
+```
```

Comment:
Could you please add one brief example using it to visualize an iterations file from one of the tutorials?

Reply:
Done, see last commit.